### PR TITLE
Adds new canister sprites

### PR DESCRIPTION
**Pull request by nicbn**
[Changelogs]:
:cl: nicbn
imageadd: Canister sprites for the new gases
/:cl:

### CODE REVIEW

**Comment by Armhulen** [comment]
POST PICS

**Comment by nicbn** [comment]
The bot already did.

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -318,7 +318,7 @@
 	var/stim_energy_change
 	stim_energy_change =heat_scale + (STIMULUM_FIRST_RISE(heat_scale**2)) - (STIMULUM_FIRST_DROP(heat_scale**3)) + (STIMULUM_SECOND_RISE(heat_scale**4)) - (STIMULUM_ABSOLUTE_DROP(heat_scale**5))
 
-	ASSERT_GAS(/datum/gas/stimulum,air)
+	ASSERT_GAS(/datum/gas/stimulum, air)
```
Comment (praisenarsie) [inline code/modules/atmospherics/gasmixtures/reactions.dm]: what's with the space here

Comment (nicbn) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: I didn't revert the whitespace changes and I'm not going to.

Comment (praisenarsie) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: there's not meant to be a space there

Comment (tortellinitony) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: it shouldn't make a difference

Comment (DoctoreMad) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: "Shouldn't" as a key word

Comment (praisenarsie) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: better safe than sorry

Comment (nicbn) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: It doesn't make difference

**Comment by Armhulen** [comment]
wow that bot is really neat

**Comment by WJohn** [comment]
These aren't terribly pretty but I guess that's more that our old canister sprites aren't great. They could be fairly easily refurbished (tweaked colors, moving the lighting to the top, general cleanup).

**Comment by Arianya** [comment]
Green would probably look better as dark green rather then neon green, and I don't think we have a canister already in that colour?

**Comment by nicbn** [comment]
We don't. And it is neon green because it is radioactive and it is like **dangeiroos**. And I'm not doing a complete canister resprite in this PR, but I can try tweaking them and making them greyscale later on.

**Comment by WJohn** [comment]
i dunno, i think a yellow/black canister like our empty canisters, but with a clear radiation symbol on it instead would be pretty good, too.

**Comment by nicbn** [comment]
I think it would make the canister too visually polluted to have a radiation symbol.

**Comment by duncathan** [comment]
These are good enough for now; if wjohn wants a full facelift for canisters it can be done later

**Comment by duncathan** [comment]
@nicbn conflicts

**Comment by nicbn** [comment]
@duncathan

File: code/modules/atmospherics/machinery/portable/canister.dm [hunk 2]
```diff
@@ -98,26 +103,31 @@
...
 /obj/machinery/portable_atmospherics/canister/nob
 	name = "hyper-noblium canister"
 	desc = "Hyper-Noblium. More noble than all other gases."
+	icon_state = "freon"
 	gas_type = /datum/gas/hypernoblium
 
 /obj/machinery/portable_atmospherics/canister/nitryl
 	name = "nitryl canister"
 	desc = "Nitryl gas. Feels great 'til the acid eats your lungs."
+	icon_state = "brown"
```
Comment (duncathan) [inline code/modules/atmospherics/machinery/portable/canister.dm]: this line appears to be causing a compilation failure

**Review by duncathan — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/machinery/portable/canister.dm (hunk 2 above)
Comment (nicbn) [reply code/modules/atmospherics/machinery/portable/canister.dm]: I already fixed it

**Pull request merged**